lib/spell.js

```diff
@@ -142,13 +151,43 @@ function parseLogicalObjectConditionValue(value) {
  *     { content: 'Diablo' },
  *   ],
  * }
+ * {
+ *   title: {
+ *     $or: [
+ *       'Leah',
+ *       'Diablo',
+ *     ]
+ *   }
+ * }
+ * {
+ *   title: {
+ *     $or: [
+ *       'Leah',
+ *       {
+ *         $like: '%jjj'
+ *       },
+ *     ]
+ *   }
+ * }
+ * {
+ *   title: {
+ *     $not: [
+ *       'Leah',
+ *       'jss'
+ *     ]
+ *   }
+ * }
  * @param {string} name logical operators, such as `$or`, `$and`
  * @param {Object|Object[]} value logical operands
  */
 function parseLogicalObjectCondition(name, value) {
   const operator = LOGICAL_OPERATOR_MAP[name];
   const conditions = parseLogicalObjectConditionValue(value);
   const args = conditions.reduce((res, condition) => {
+    if (!isPlainObject(condition)) return res.concat({
+      type: 'literal',
+      value: condition,
+    });
```

Comment:
multi line if/else branches should be wrapped in `{}`

```suggestion
    if (!isPlainObject(condition)) {
      return res.concat({
        type: 'literal',
        value: condition,
      });
    }
```